apps/server/src/endpoints.ts

```diff
@@ -86,10 +88,20 @@ export const createGame =
       switch (body.modelType) {
         case ModelType.THREAT_DRAGON: {
           // TODO: validation
-          await gameServer.db.setModel(
-            matchID,
-            JSON.parse(body.model as string) as ThreatDragonModel,
-          );
+          logEvent('vishal printing body');
```

Comment:
Please remove the log statements and the ones below (lines 100 and 101), please

Reply:
sure